### DISCUSSION TITLE
Pico2 PCB Design Files

### DISCUSSION THREAD

**Discussion by BusPirateV5** [gusmanb/logicanalyzer#172] [Q&A]
Can you please provide a link directly to the PCB design files for the latest version of the Pico2 that is meant to be used with the 24 Channel Logic Analyzer by GusmanB?

I have received a lot of people providing reference, but can someone please provide a direct link?

Thank your for your time and consideration! Have a wonderful day and Happy Holidays!

**Comment by rdfry** [comment]
I am also interested in your project
